platforms/defaults_windows.go

```diff
@@ -39,25 +39,28 @@ func DefaultSpec() specs.Platform {
 	}
 }
 
-type matchComparer struct {
-	defaults        Matcher
+type windowsmatcher struct {
+	specs.Platform
 	osVersionPrefix string
+	defaultMatcher  Matcher
 }
 
 // Match matches platform with the same windows major, minor
 // and build version.
-func (m matchComparer) Match(p imagespec.Platform) bool {
-	if m.defaults.Match(p) {
-		// TODO(windows): Figure out whether OSVersion is deprecated.
-		return strings.HasPrefix(p.OSVersion, m.osVersionPrefix)
+func (m windowsmatcher) Match(p imagespec.Platform) bool {
```

Comment:
I realise this isn't part of this change, but is there any reason to have `imagespec` _and_ `specs` in this file? They appear to reference the same module, and the only uses of `imagespec` are both touched by this commit, so maybe now's the time to replace the two uses of `imagespec` with `specs`.

Or perhaps there's actually a distinction I'm overlooking.

Reply:
I don't think you're overlooking anything, and I've updated to use specs with is used in the other files